scripts/wattlab_parity_diff.py

```
@@ -0,0 +1,438 @@
+        r.get("unoccupied_hours")
+        for r in ofdd_rows
+        if isinstance(r, dict) and "unoccupied_hours" in r
+    ]
+    all_zero_unocc = bool(unocc) and all(
+        isinstance(u, (int, float)) and float(u) == 0.0 for u in unocc
+    )
+    rows.append(
+        {
+            "artifact": "schedule_analytics",
+            "key": "engine_definition",
+            "vibe19": "OccupancySchedule calendar mask (07–17 M–F)",
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)